### PR TITLE
feat(starr anime): updates to various CFs

### PR DESCRIPTION
**Pull request by FonduemangVI**
# Pull Request

## Purpose

Updates to various Anime CFs

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: docs/json/radarr/cf/anime-lq-groups.json [hunk 1]
```diff
@@ -636,6 +636,15 @@
         "value": "\\b(MinisCuba)\\b"
       }
     },
+    {
+      "name": "MiniMTBB",
```
Comment (nuxencs) [inline docs/json/radarr/cf/anime-lq-groups.json:640]: sorting is off, should be before `MinisCuba`

File: docs/json/sonarr/cf/anime-lq-groups.json [hunk 2]
```diff
@@ -636,6 +636,15 @@
         "value": "\\b(MinisCuba)\\b"
       }
     },
+    {
+      "name": "MiniMTBB",
```
Comment (nuxencs) [inline docs/json/sonarr/cf/anime-lq-groups.json:640]: sorting is off, should be before `MinisCuba`

**Review by nuxencs — DISMISSED** [review]
just two small changes, otherwise looks good 😄

File: docs/json/radarr/cf/dubs-only.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
       "negate": false,
       "required": false,
       "fields": {
-        "value": "(?<!multi-)\\b(dub(bed)?)\\b|(funi|eng(lish)?)_?dub"
+        "value": "^(?!.*Dual[-_. ]?Audio).*((?<!multi-)\\b(dub(bed)?)\\b|(funi|eng(lish)?)_?dub)"
```
Comment (rg9400) [inline docs/json/radarr/cf/dubs-only.json:15]: change the regex to "^(?!.*(Dual|Multi)[-_. ]?Audio).*((?<!multi-)\\b(dub(bed)?)\\b|(funi|eng(lish)?)_?dub)" to handle the mutli-audio dub cases (Arg0 Solo Levelling example)

File: docs/json/sonarr/cf/dubs-only.json [hunk 4]
```diff
@@ -12,7 +12,7 @@
       "negate": false,
       "required": false,
       "fields": {
-        "value": "(?<!multi-)\\b(dub(bed)?)\\b|(funi|eng(lish)?)_?dub"
+        "value": "^(?!.*Dual[-_. ]?Audio).*((?<!multi-)\\b(dub(bed)?)\\b|(funi|eng(lish)?)_?dub)"
```
Comment (rg9400) [inline docs/json/sonarr/cf/dubs-only.json:15]: change the regex to `"^(?!.*(Dual|Multi)[-_. ]?Audio).*((?<!multi-)\\b(dub(bed)?)\\b|(funi|eng(lish)?)_?dub)"` to handle the mutli-audio dub cases (Arg0 Solo Levelling example)

File: docs/json/radarr/cf/anime-lq-groups.json (hunk 1 above)
Comment (FonduemangVI) [reply docs/json/radarr/cf/anime-lq-groups.json:640]: Fixed

File: docs/json/sonarr/cf/anime-lq-groups.json (hunk 2 above)
Comment (FonduemangVI) [reply docs/json/sonarr/cf/anime-lq-groups.json:640]: Fixed

File: docs/json/radarr/cf/dubs-only.json (hunk 3 above)
Comment (FonduemangVI) [reply docs/json/radarr/cf/dubs-only.json:15]: Fixed

File: docs/json/sonarr/cf/dubs-only.json (hunk 4 above)
Comment (FonduemangVI) [reply docs/json/sonarr/cf/dubs-only.json:15]: Fixed

**Review by rg9400 — APPROVED** [review]
Looks good to me

**Pull request merged**